src/poll.rs

```diff
@@ -537,22 +543,23 @@ struct AtomicState {
 
 const MASK_2: usize = 4 - 1;
 const MASK_4: usize = 16 - 1;
+const MASK_8: usize = 256 - 1;
 const QUEUED_MASK: usize = 1 << QUEUED_SHIFT;
 const DROPPED_MASK: usize = 1 << DROPPED_SHIFT;
 
 const READINESS_SHIFT: usize = 0;
-const INTEREST_SHIFT: usize = 4;
-const POLL_OPT_SHIFT: usize = 8;
-const TOKEN_RD_SHIFT: usize = 12;
-const TOKEN_WR_SHIFT: usize = 14;
-const QUEUED_SHIFT: usize = 16;
-const DROPPED_SHIFT: usize = 17;
+const INTEREST_SHIFT: usize = 8;
+const POLL_OPT_SHIFT: usize = 16;
+const TOKEN_RD_SHIFT: usize = 20;
+const TOKEN_WR_SHIFT: usize = 22;
+const QUEUED_SHIFT: usize = 24;
+const DROPPED_SHIFT: usize = 25;
```

Comment:
I'm not using AIO in the custom readiness queue, but I think it might be useful on Linux.  Linux's epoll doesn't support AIO operations, so the easiest way to make them work with mio would be to associate a signalfd with the libaio requests and register that signalfd with mio for Readable.  Whenever something arrives in the signalfd, you would parse the struct and then set an Aio event in the custom readiness queue.  Do you still think I should remove this part?